docs/userguide/application.rst

```diff
@@ -333,7 +333,7 @@ Finalizing the object will:
 
     #. Make sure all tasks are bound to the current app.
 
-        Tasks are bound to apps so that it can read default
+        Tasks are bound to an app so that they can read default
         values from the configuration.
```

Comment:
'they' in this sentence is referring to 'Tasks', yes? i.e., 'Tasks can read default values from configuration?